dbms/src/Debug/MockSSTReader.h

```diff
@@ -80,13 +80,13 @@ struct MockSSTReader
     {
         if (et == EngineIteratorSeekType::First)
         {
-            remained = iter != end;
             iter = begin;
+            remained = iter != end;
         }
         else if (et == EngineIteratorSeekType::Last)
         {
-            remained = iter != end;
             iter = end;
+            remained = iter != end;
```

Comment:
I think this means always `remained = false`?

Reply:
fixed